specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-02-01-preview/imagebuilder.json

```diff
@@ -0,0 +1,1148 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "VirtualMachineImageTemplate",
+    "description": "Virtual Machine Image Template",
+    "version": "2019-02-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.VirtualMachineImages/imageTemplates": {
```

Comment:
For hygiene, naming should be Microsoft.virtualMachineImages....

Reply:
Not sure I understand... do you mean small 'v' instead of capital 'V'? or is there another type I am missing?

Reply:
Please disregard. This is fine for RP namespace. PEBKAC